File: services/device/src/main/java/com/example/iter/device/dto/response/EquipmentEstimateResponse.java
package com.example.iter.device.dto.response;

import java.math.BigDecimal;
import java.time.LocalDate;

public record EquipmentEstimateResponse(
        Long equipmentId,
        LocalDate startDate,
        LocalDate endDate,
        int rentalDays,
        BigDecimal dailyPrice,
        BigDecimal totalPrice
) {
}

File: services/device/src/main/kotlin/iter/device/dto/response/EquipmentEstimateResponse.kt
package iter.device.dto.response

import java.math.BigDecimal
import java.time.LocalDate

data class EquipmentEstimateResponse(
    val equipmentId: Long,
    val startDate: LocalDate,
    val endDate: LocalDate,
    val rentalDays: Int,
    val dailyPrice: BigDecimal,
    val totalPrice: BigDecimal,
)
